site/content/usage/13-iamserviceaccounts.md

```diff
@@ -34,6 +34,8 @@ The IAM OIDC Provider is not enabled by default, you can use the following comma
 eksctl utils associate-iam-oidc-provider --cluster=<clusterName>
 ```
 
+By default the Client ID list (audience) of IAM OIDC Provider will be set to `sts.amazonaws.com`. To override it you can specify `--oidc-client-id-list` multiple times.
```

Comment:
I think if the flag is a list you should be able to just provide it once like: `--oidc-client-id-list sts1.amazon.com,sts.us-east-1.amazonaws.com,sts.eu-west-1.amazonaws.com` 

Can you do a little manual testing to check this and update the comment?

Reply:
I have tested it using a comma-separated list and it works, but I have seen similar explanation in the docs:
```
NOTE: you can specify `--attach-policy-arn` multiple times to use more then one policy.
``` 

should I change it?